project-keeper/src/main/resources/templates/.github/workflows/dependencies_check.yml

```diff
@@ -48,12 +48,25 @@ jobs:
         env:
           CREATED_ISSUES: ${{ steps.security-issues.outputs.created-issues }}
 
+      - name: Report Status to Slack channel
+        if: always()
+        uses: ravsamhq/notify-slack-action@v2
+        with:
+          status: ${{ job.status }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          notification_title: "Dependency check in {repo} has {status_message}"
+          message_format: "{emoji} *{workflow}* {status_message} in <{repo_url}|{repo}>"
+          notify_when: "failure,cancelled,warnings,skipped"
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.INTEGRATION_TEAM_SLACK_NOTIFICATION_WEBHOOK }}
+
   start_dependency_udpate:
```

Comment:
When deploying PK to many java projects, PK will add (enforce) these templates to be used in each of the many java projects and dependency checks will run regularly, say daily.
I am a little bit concerned, that in case of a central issue in dependency check process, the slack channel might get flooded. Should we think about a mitigation, here?

Reply:
OK, I will remove the notification for failures